packages/bot-web-ui/src/stores/journal-store.js

```diff
@@ -30,7 +37,9 @@ export default class JournalStore {
     ];
 
     @observable is_filter_dialog_visible = false;
-    @observable unfiltered_messages = [];
+    @observable unfiltered_messages = Array.isArray(JSON.parse(sessionStorage.getItem(journal_storage_key)))
+        ? JSON.parse(sessionStorage.getItem(journal_storage_key))
+        : [];
```

Comment:
We're calling `JSON.parse(sessionStorage.getItem(journal_storage_key))` here twice, can we call it only once?